NLog.Web.Tests/LayoutRenderers/AspNetSessionValueLayoutRendererTests.cs

```diff
@@ -23,6 +24,7 @@ public AspNetSessionValueLayoutRendererTests()
 
         public void SetUp()
         {
+            ConfigurationItemFactory.Default.LayoutRenderers.RegisterDefinition("aspnet-session", typeof(AspNetSessionValueLayoutRenderer));
```

Comment:
@petemounce I added some comments there. Not sure if you have seen them. If not, I'm not in a hurry, so NP